SUPPORT.md

```diff
@@ -1 +1,24 @@
-See our [README.md](README.md) for ways to get help with AMP.
+# Getting Support	See our [README.md](README.md) for ways to get help with AMP.
```

Comment:
Just noticed that you probably want to delete the second part here.

Reply:
And could you add a comment about this dependency between the website and this repro?